internal/pkg/deploy/cloudformation/project.go

```diff
@@ -63,6 +63,70 @@ func (cf CloudFormation) CreateProjectResources(project *archer.Project) error {
 	return nil
 }
 
+// GetProjectResourcesByRegion fetches all the regional resources for a particular region.
+func (cf CloudFormation) GetProjectResourcesByRegion(project *archer.Project, region string) (*archer.ProjectRegionalResources, error) {
+	resources, err := cf.getResourcesForStackInstances(project, &region)
+	if err != nil {
+		return nil, fmt.Errorf("describing project resources: %w", err)
+	}
+	if len(resources) == 0 {
+		return nil, fmt.Errorf("no regional resources for project %s in region %s found", project.Name, region)
+	}
+
+	return resources[0], nil
+}
+
+// GetRegionalProjectResources fetches all the regional resources for a particular project.
+func (cf CloudFormation) GetRegionalProjectResources(project *archer.Project) ([]*archer.ProjectRegionalResources, error) {
```

Comment:
nit: probably should just name this to `GetProjectResources` because initially I thought this function duplicates the one above with a different name.

Reply:
Ah, this name. I namespaced them is because we also have global project resources (the roles for the project).